flutter-studio/src/io/flutter/actions/OpenFlutterNewsSidePanelAction.java

```diff
@@ -124,7 +124,7 @@ public void toolWindowRegistered(@NotNull String id) {
 
     @Override
     public void toolWindowUnregistered(@NotNull String id, @NotNull ToolWindow toolWindow) {
-      if (id.equals(OpenAssistSidePanelAction.TOOL_WINDOW_TITLE)) {
+      if (id.equals("Assistant:")) {
```

Comment:
Do you want to make this a constant too?

Reply:
No, this code isn't even used.